### PR TITLE
Install all Python dependencies from PyPI

### LINKED ISSUE #9: Remove Python venv building script
This is currently blocked by the following upstream dependencies:

- ~~`libecl`~~
- `libres`
- `opm`

### PR DESCRIPTION
**Pull request by anders-kiaer**
*Insert a description of your pull request (PR) here, and check off the boxes below when they are done.*

---

### Contributor checklist

- [X] :tada: This PR closes #9.
- [x] :scroll: I have broken down my PR into the following tasks:
   - [X] Remove Python venv building script
   - [x] Add dependencies as formal dependencies in `setup.py`
   - [x] Make `FORWARD_MODEL` `Flow` job installation automatic.
- [x] :book: I have considered adding a new entry in `CHANGELOG.md`.
- [x] :books: I have considered updating the documentation.

### CODE REVIEW

File: .github/workflows/flownet.yml [hunk 1]
```diff
@@ -56,35 +50,28 @@ jobs:
 
       - name: 📦 Install FlowNet
         run: |
-          source $VENV_PATH/bin/activate
+          pip install "matplotlib<3.2"  # As long as ERT requires it...
```
Comment (wouterjdb) [inline .github/workflows/flownet.yml:53]: Should we document this in the Readme as well? Or shall we assume the user will understand the error?

File: src/flownet/ert/FLOW_SIMULATION [hunk 2]
```diff
@@ -0,0 +1,9 @@
+EXECUTABLE    flownet_run_flow
```
Comment (wouterjdb) [inline src/flownet/ert/FLOW_SIMULATION:1]: What's up with the spaces in this file?

File: src/flownet/templates/common_config.ert.jinja2 [hunk 3]
```diff
@@ -33,4 +33,4 @@ FORWARD_MODEL COPY_FILE(<FROM>=<CONFIG_PATH>/PROPS.inc, <TO>=./eclipse/model/inc
 FORWARD_MODEL COPY_FILE(<FROM>=<CONFIG_PATH>/SOLUTION.inc, <TO>=./eclipse/model/include/SOLUTION.inc)
 FORWARD_MODEL COPY_FILE(<FROM>=<CONFIG_PATH>/SCHEDULE.inc, <TO>=./eclipse/model/include/SCHEDULE.inc)
 
-SIMULATION_JOB {{ config.ert.simulator.name.upper() }} <ECLBASE> {%- if config.ert.simulator.version is not none: -%}"--version={{ config.ert.simulator.version }}"{%- endif %}
+FORWARD_MODEL FLOW_SIMULATION(<DATA_FILE>=<ECLBASE>)
```
Comment (wouterjdb) [inline src/flownet/templates/common_config.ert.jinja2:36]: Maybe remove `config.ert.simulator.version` completely from the config?

**Review by wouterjdb — CHANGES_REQUESTED** [review]

File: .github/workflows/flownet.yml (hunk 1 above)
Comment (anders-kiaer) [reply .github/workflows/flownet.yml:53]: I think we can simply assume the user to understand for now. Shouldn't be more than a couple of weeks before this is resolved in PyPI ERT-version (https://github.com/equinor/ert/pull/1149)

File: src/flownet/templates/common_config.ert.jinja2 (hunk 3 above)
Comment (anders-kiaer) [reply src/flownet/templates/common_config.ert.jinja2:36]: Good question - copied from `semeio`. I'll remove some of them.

File: src/flownet/ert/FLOW_SIMULATION (hunk 2 above)
Comment (anders-kiaer) [reply src/flownet/ert/FLOW_SIMULATION:1]: Good question - copied from `semeio`. I'll remove some of them.

**Review by wouterjdb — APPROVED** [review]
LGTM 🚀

**Pull request merged**